addon/services/device/layout.js

```diff
@@ -101,7 +103,9 @@ export default Service.extend({
 
     if (typeof window === 'object' && typeof document === 'object') {
       this.setupResize();
-      this.updateResolution();
+    if (!this.get('isDestroyed') || !this.get('isDestroying')) {
```

Comment:
I don't see any async here, and AFAIK it is impossible to be destroying or destroyed inside of init...